aptos-move/framework/aptos-framework/sources/voting.move

```diff
@@ -189,6 +190,7 @@ module aptos_framework::voting {
     }
 
     public fun register<ProposalType: store>(account: &signer) {
+        permissioned_signer::assert_master_signer(account);
```

Comment:
Could you explain why this assertion is added here and nowhere else?

Reply:
That was the only function that has signer as argument?